README.md

```diff
@@ -35,17 +43,17 @@ Rscript -e 'library(devtools);install_github("USGS-R/gsplot")'
 Rscript -e 'library(devtools);install_url("https://github.com/USGS-R/repgen/archive/'${release_version}'.zip")'
 ```
 
-Example usgage
-----------
-###Generate an 'extremes' report:
+## Example usgage
+
+### Generate an 'extremes' report:
 ```R
 library(repgen)
 library(jsonlite)
 
 data <- fromJSON(system.file('extdata','extremes','extremes-example.json',package = 'repgen'))
 extremes(data, 'pdf')
```

Comment:
i would probably change 'pdf' to 'html' since we don't support pdf anymore actually...